NBitcoin.Altcoins/HashX11/X13.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
```

Comment:
I thought we already had X13?

Reply:
I didn't see it anywhere, unless I missed it somewhere else in NBitcoin?